src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -51,6 +51,7 @@ export interface Model3DViewerProps {
   boundingBox?: THREE.Box3;
   cache?: CacheObject;
   enableKeyboardNavigation?: boolean;
+  highlightNodesOnComplete?: boolean;
```

Comment:
```suggestion
  highlightNodes?: boolean;
```
but this basically means that nodes will never be highlighted?
so than maybe call it as just `highlightNodes`?

Reply:
you should disable node highlighting in general (with this property). So better to move `highlightNodes` prop checking inside `highlightNodes` private method (you should change method name first)

Reply:
Implemented these suggestions - but kept the private method name as it doesn't conflict with the same named prop name